projects/storefrontlib/src/cms-components/product/product-images/product-images.component.html

```diff
@@ -2,14 +2,16 @@
   <cx-media [container]="main" format="zoom"> </cx-media>
 </ng-container>
 
-<cx-carousel
-  *ngIf="!isThumbsEmpty"
-  class="thumbs"
-  [items]="thumbs$ | async"
-  itemWidth="120px"
-  [hideIndicators]="false"
-  [template]="thumb"
-></cx-carousel>
+<ng-container *ngIf="thumbs$ | async as thumbs">
+  <cx-carousel
+    *ngIf="thumbs.length"
```

Comment:
If we plan to not use `isThumbsEmpty` property anymore we should consider deprecating it in a component.